sharding-core/src/main/java/in/cleartax/dropwizard/sharding/transactions/TransactionRunner.java

```diff
@@ -53,6 +56,8 @@ public T start(boolean reUseSession, UnitOfWork unitOfWork) throws Throwable {
             aspect.onFinish();
             DelegatingTenantResolver.getInstance().setDelegate(null);
         }
+        log.info("[DATABASE] transaction={} error={} operation={} time-elapsed={}",
+                unitOfWork.transactional(), ex != null, operation, System.currentTimeMillis() - startTime);
```

Comment:
Log it in seconds?

Reply:
Logging it in seconds will lead to less accurate instrumentation.